app/views/layouts/_messages.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+<% flash.each do |name, msg| %>
+    <div class="alert alert-<%= name == "notice" ? "success" : "danger" %> alert-dismissible fade show" role="alert">
```

Comment:
`name == "notice" ? "success" : "danger"` Please move this code to helper method